lib/mayaUsd/render/vp2RenderDelegate/draw_item.h

```diff
@@ -125,6 +125,16 @@ class HdVP2DrawItem final : public HdDrawItem
         kSelectionHighlight = kRegular << 1 //!< Selection highlight.
     };
 
+    //! Flags specifying mod properties
+    enum ModFlags
```

Comment:
Just curious: mod stands for modifier?

Reply:
Mod is a draw item which serves as a modification of the original/basic draw item.